cookbook/form/data_transformers.rst

```diff
@@ -16,24 +16,24 @@ to render the form, and then back into a ``DateTime`` object on submit.
     When a form field has the ``inherit_data`` option set, Data Transformers
     won't be applied to that field.
 
-Simple Example: Sanitizing HTML on User Input
----------------------------------------------
+Simple Example: Transforming tags string from User Input to array
+-----------------------------------------------------------------
 
-Suppose you have a Task form with a description ``textarea`` type::
+Suppose you have a Task form with a tags ``text`` type::
 
     // src/AppBundle/Form/TaskType.php
     namespace AppBundle\Form\Type;
 
     use Symfony\Component\Form\FormBuilderInterface;
     use Symfony\Component\OptionsResolver\OptionsResolver;
-    use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
 
     // ...
     class TaskType extends AbstractType
     {
         public function buildForm(FormBuilderInterface $builder, array $options)
         {
-            $builder->add('description', TextareaType::class);
+            $builder->add('tags', TextType::class);
```

Comment:
I agree that we should merge this into the `2.3` branch first. You don't need to create a new PR for this (we can rebase when merging), but can you please change the type here to be simply `text` (we will then update the type when merging things up)?